README.md

```diff
@@ -10,8 +10,8 @@
 
 Works as a valid Helm Chart Repository, and also provides an API for uploading charts.
 
-<img width="120" align="right" src="https://github.com/golang-samples/gopher-vector/raw/master/gopher-side_color.png">
-<img width="40" align="right" src="https://github.com/golang-samples/gopher-vector/raw/master/gopher-side_color.png">
+<img width="120" align="right" src="https://github.com/redblue9771/gopher-vector/raw/master/gopher-side_color.png">
```

Comment:
ahh... I think it is my network issue until you fix this

Reply:
I think the original repo was removed from github